src/components/cards/employees-card/index.js

```diff
@@ -0,0 +1,2 @@
+import EmployeeCard from './EmployeeCard';
+export default EmployeeCard;
```

Comment:
You can put an EOL character at the end
To make it automatic you can install a plugin on your IDE to read the `.editorconfig` file